inc/Provider.php

```diff
@@ -220,7 +222,7 @@ final public function remote_request( string $url, array $args ) : string {
 			);
 		}
 
-		return $response_body;
+		return $response['http_response'];
```

Comment:
Is this always set?

Reply:
Yep, at least at this stage - the method throws earlier if it's a `WP_Error`.